cpp/include/raft/neighbors/brute_force.cuh

```diff
@@ -138,13 +138,15 @@ inline void knn_merge_parts(
  * 					 is ignored if the metric_type is not Minkowski.
  * @param[in] global_id_offset: optional starting global id mapping for the local partition
  *                              (assumes the index contains contiguous ids in the global id space)
+ * @param[in] distance_epilogue: optional epilogue function to run after computing distances
```

Comment:
Can you add the expectation if the epilogue function's argument list here too? Just a small example prototype of the function definition will do.